src/components/NcActions/NcActions.vue

```diff
@@ -725,6 +725,16 @@ export default {
 			type: Number,
 			default: 0,
 		},
+
+		/**
+		 * Enable if actions are website navigation links
+		 *
+		 * Removes attributes incompatible with accessible navigation
+		 */
+		isNav: {
+			type: Boolean,
+			default: false,
+		},
```

Comment:
It can probably be done automatically by checking the domain?
I'm scared of the amount of props we have everywhere :see_no_evil: